packages/components/src/composite/stories/legacy.story.tsx

```diff
@@ -0,0 +1,244 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn, StoryContext } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { isRTL } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Composite,
+	CompositeItem,
+	CompositeGroup,
+	useCompositeState,
+} from '../legacy';
+
+import type { InitialState } from '../legacy';
+
+const Placeholder = ( _: InitialState ) => <></>;
+
+const transform = ( code: string, context: StoryContext ) => {
+	const config = ` ${ JSON.stringify( context.args, null, 2 ) } `;
+	const state = config.replace( ' {} ', '' );
+	return [
+		`const state = useCompositeState(${ state });`,
+		'',
+		'return (',
+		'  ' +
+			code
+				.replaceAll( /state=\{\{[\s\S]*?\}\}/g, 'state={ state }' )
+				.replaceAll( '=>', '' )
+				.replaceAll( /baseId=[^>]+?(\s*>)/g, ( _, close ) => {
+					return `{ ...state }${ close }`;
+				} )
+				.replaceAll( /<Composite\w+[\s\S]*?>/g, ( match ) =>
+					match.replaceAll( /\s+\s/g, ' ' )
+				)
+				.replaceAll(
+					/ >\s+([\w\s]*?)\s+<\//g,
+					( _, value ) => `>${ value }</`
+				)
+				.replaceAll( '} >', '}>' )
+				.replaceAll( /\n/g, '\n  ' ),
+		');',
+	].join( '\n' );
+};
+
+const meta: Meta< typeof Placeholder > = {
+	title: 'Components/Composite/Composite (Legacy)',
+	id: 'components-composite-legacy',
+	component: Placeholder,
+	args: {
+		rtl: isRTL() || undefined,
```

Comment:
Is passing the the `rtl` argument explicitly necessary for this component to work correctly in Storybook when simulating RTL writing directions? (cc @mirka)

Reply:
No, it's not really necessary, and actually misleading here, as it hides the fact that consumers need to explicitly set it themselves. Will remove.